### PR TITLE
Update Cloud Memorystore Redis API config

### PR DESCRIPTION
**Pull request by jskeet**
When this is merged, I'll do a release.

### CODE REVIEW

File: apis/Google.Cloud.Redis.V1Beta1/Google.Cloud.Redis.V1Beta1/Google.Cloud.Redis.V1Beta1.csproj [hunk 1]
```diff
@@ -5,13 +5,12 @@
...
     <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netstandard1.5</TargetFrameworks>
     <LangVersion>latest</LangVersion>
-    <Features>IOperation</Features>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <AssemblyOriginatorKeyFile>../../GoogleApis.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
     <Deterministic>true</Deterministic>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <Description>Recommended Google client library to manage Redis.</Description>
+    <Description>Recommended Google client library to administer Cloud Memorystore Redis.</Description>
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Redis.V1Beta1/Google.Cloud.Redis.V1Beta1/Google.Cloud.Redis.V1Beta1.csproj]: "Cloud Memorystore *for* Redis" ?
https://cloud.google.com/memorystore/

File: apis/apis.json [hunk 2]
```diff
@@ -470,9 +470,11 @@
 
   {
     "id": "Google.Cloud.Redis.V1Beta1",
+    "productName": "Google Cloud Memorystore Redis",
```
Comment (chrisdunelm) [inline apis/apis.json]: OK, probably here really

File: apis/apis.json [hunk 3]
```diff
@@ -470,9 +470,11 @@
...
+    "productUrl": "https://cloud.google.com/memorystore/",
     "version": "1.0.0-beta01",
     "type": "grpc",
-    "description": "Recommended Google client library to manage Redis.",
+    "description": "Recommended Google client library to administer Cloud Memorystore Redis.",
```
Comment (chrisdunelm) [inline apis/apis.json]: ...and here

File: apis/apis.json (hunk 2 above)
Comment (jskeet) [reply apis/apis.json]: This was the name given to me by the team, so I'll stick with it for now.

Comment (jskeet) [reply apis/apis.json]: (Except that that's not what the web site says, so I'll match that.)

**Pull request merged**